docs/ecosystem/defi-liquidity/faq.md

```diff
@@ -23,6 +23,19 @@ import Details from '@theme/Details';
 
 Below are common questions regarding stablecoins, liquidity, and bridging on Flow. Click on each question to expand and view the answer.
 
+## Bridging and Gas
+
+<Details summary="Does Flow use ETH for gas?">
+No, Flow uses $FLOW as the gas token. [WETH](./defi-contracts#stablecoins--wrapped-assets) is supported on Flow EVM when [bridging](../bridges.md) from another chain.
```

Comment:
Would add a note: Also note, $WFLOW is used as an ERC20 in DeFi apps and not used for gas.

Reply:
```suggestion
No, Flow uses $FLOW as the gas token. [WETH](./defi-contracts#stablecoins--wrapped-assets) is supported on Flow EVM when [bridging](../bridges.md) from another chain. $WFLOW is used as an ERC20 in DeFi apps and not used for gas.
```